composer/utils/checkpoint.py

```diff
@@ -36,7 +35,7 @@
 
 _COMPOSER_STATES_FILENAME = 'composer_states.pt'
 _DEEPSPEED_TAG = 'deepspeed'  # always tag with the same, deterministic name. We'll rename the tarball to the appropriate name.
-
+_TORCH_DISTRIBUTED_CHECKPOINTS_FILENAME = f"__{dist.get_global_rank()}_0.distcp"
 
```

Comment:
putting `{dist.get_global_rank()}` will make it fail if dist is not initialized, which is a problem for "normal" non-distributed usage

Reply:
pretty sure it just returns 0 if dist is not initialized